extension/src/test/java/org/camunda/bpm/extension/keycloak/test/AbstractKeycloakIdentityProviderTest.java

```diff
@@ -168,6 +169,8 @@ protected static KeycloakIdentityProviderPlugin configureKeycloakIdentityProvide
 				kcp.setKeycloakAdminUrl(kcp.getKeycloakAdminUrl().replace("https://localhost:9001/auth", KEYCLOAK_URL));
 				kcp.setKeycloakIssuerUrl(kcp.getKeycloakIssuerUrl().replace("https://localhost:9001/auth", KEYCLOAK_URL));
 				kcp.setClientSecret(CLIENT_SECRET);
+				kcp.setCacheEnabled(true);
+				kcp.setCustomHttpRequestInterceptors(Collections.singletonList(new CountingHttpRequestInterceptor()));
```

Comment:
@juja0:

Instead of activating the cache for all tests - which you actually do by modifying the abstract test class - I would expect the following:
- Cache is deactivated by default - it should be an active decision of the user to switch it on
- All existing unit tests are running just like they have been running before: without cache activated
- There is an extra test case with a separate configuration.xml
-- showing all configuration options (activate cache, configure cache size and timeout both with a non default value)
-- testing the cache explicitely

Testing should separate the caching concern from other aspects and query tests.

Reply:
Makes sense. Will work on this over the weekend.

Reply:
Done